dataactbroker/handlers/fileHandler.py

```diff
@@ -551,4 +551,16 @@ def checkD2File(self, submission_id=None):
         response = {"status": status, "url": url, "start": str(d2_file.start_date), "end": str(d2_file.end_date),
                     "message": error_message}
 
-        return JsonResponse.create(StatusCode.OK, response)
\ No newline at end of file
+        return JsonResponse.create(StatusCode.OK, response)
+
+    def getProtectedFiles(self):
+        """ Returns a set of urls to protected files on the help page """
+        response = {}
+        response["message"] = ""
+        if self.isLocal:
+            response["urls"] = {}
+            response["message"] = "Cannot use get_protected_files route locally"
+            return JsonResponse.create(StatusCode.CLIENT_ERROR, response)
+
+        response["urls"] = self.s3manager.getFileUrls(bucket_name=CONFIG_BROKER["static_files_bucket"], path=CONFIG_BROKER["help_files_path"])
```

Comment:
Is `static_files_bucket` a new entry in `config.yml`? If so, we should add it to `config_example.yml`.